main.js

```diff
@@ -99,15 +99,25 @@ var eventNameCapitalization = {
     onselectionchanging: "onSelectionChanging",
     onsplittoggle: "onSplitToggle",
     onsuggestionsrequested: "onSuggestionsRequested",
-    onzoomchanged: "onZoomChanged"
+    onzoomchanged: "onZoomChanged",
+
+    onafterhidecontrols: "onAfterHideControls",
+    onaftershowcontrols: "onAfterShowControls",
+    onbeforehidecontrols: "onBeforeHideControls",
+    onbeforeshowcontrols: "onBeforeShowControls",
+    onmarkerreached: "onMarkerReached",
+    onmediacommandexecuted: "onMediaCommandExecuted",
+    ontargetratechange: "onTargetRateChange",
+    ontargettimeupdate: "onTargetTimeUpdate",
+    onthumbnailrequest: "onThumbnailRequest"
```

Comment:
nit: Instead of putting these new events at the bottom of `eventNameCapitalization`, can you keep `eventNameCapitalization` sorted in alphabetical order? (Maybe also add a comment above `var eventNameCapitalization` to note that the events are sorted alphabetically for readability so that this convention is obvious to future maintainers)